lms/static/js/spec/learner_dashboard/collection_list_view_spec.js

```diff
@@ -29,7 +29,14 @@ define([
                             modified: '2016-03-25T13:45:21.220732Z',
                             marketing_slug: 'p_2?param=haha&test=b', 
                             id: 146,
-                            marketing_url: 'http://www.edx.org/xseries/p_2?param=haha&test=b'
+                            marketing_url: 'http://www.edx.org/xseries/p_2?param=haha&test=b',
+                            /* jshint ignore:start */
+                            banner_image_urls: [
+                                '348x116': 'http://www.edx.org/images/org1/test1',
+                                '435x145': 'http://www.edx.org/images/org1/test2',
+                                '726x242': 'http://www.edx.org/images/org1/test3'
+                            ]
+                            /* jshint ignore:end */
```

Comment:
javascript do not like number to begin for a key in a dictionary definition.
See http://stackoverflow.com/questions/3633362/is-there-anyway-to-use-a-numeric-type-as-an-object-key